test/mocks/overrides/override_pin_view_model.dart

```diff
@@ -11,6 +12,7 @@ class MockPinViewModel extends AutoDisposeAsyncNotifier<List<MapPinDetails>>
 
   MockPinViewModel({
     Future<List<MapPinDetails>> Function()? build,
+    Function(BuildContext)? setContext,
```

Comment:
You no longer need this.

Reply:
Fixed with [82d65b9](https://github.com/ProximaEPFL/proxima/pull/335/commits/82d65b9eff2d4e6a69793b06a208dede64fa5cd8)